yaksh/models.py

```diff
@@ -121,10 +121,10 @@
 
 
 def get_assignment_dir(instance, filename):
-    folder_name = instance.course.name.replace(" ", "_")
-    sub_folder_name = instance.question_paper.quiz.description.replace(
-        " ", "_")
-    return os.sep.join((folder_name, sub_folder_name, instance.user.username,
+    folder = str(instance.answer_paper.course.id)
+    sub_folder = str(instance.answer_paper.question_paper.quiz.id)
```

Comment:
Use `answer_paper.course_id` instead of `answer_paper.course.id`. Same for quiz id.